### PR TITLE
Add Py3 support

### PR DESCRIPTION
**Pull request by genme**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,7 @@
 *.pyc
 build/
 dist/
-rfc6266_parser.egg-info/
+*.egg-info/
 .tox/
+.venv/
+.python-version
```
Comment (homm) [inline .gitignore]: new line

File: setup.py [hunk 2]
```diff
@@ -2,21 +2,29 @@
...
 
 setup(
-    author='Simon Woerner',
-    author_email='git@simon-woerner.de',
-    description='Parse and generate Content-Disposition headers; Fork of rfc6266 with bugfixes.',
-    url='https://github.com/SWW13/python-rfc6266-parser',
+    author='Arseniy Baranov',
+    author_email='arseniy.baranov@gmail.com',
+    description='Fork of rfc6266 package with python 3.6+ support',
+    url='https://github.com/genme/python-rfc6266-parser',
```
Comment (homm) [inline setup.py]:
```suggestion
    url='https://github.com/uploadcare/python-rfc6266-parser',
```

File: README [hunk 3]
```diff
@@ -1,75 +1,4 @@
...
-
-* RFC 6266 <https://tools.ietf.org/html/rfc6266>
-  specifies the Content-Disposition header
-* RFC 5987 <https://tools.ietf.org/html/rfc5987>
-  specifies a way to encode non-ascii filenames
-* TC 2231 <http://greenbytes.de/tech/tc2231/>
-  is a test suite for Content-Disposition headers
-
-
+Python 3.6+ compliant fork of rfc6266-parser.
```
Comment (homm) [inline README]: NL

File: test_rfc6266_parser.py [hunk 4]
```diff
@@ -56,42 +45,25 @@ def test_location_fallback():
...
+    assert parse_headers('attachment;').disposition == 'attachment'
+    assert parse_headers('attachment; key=val;').disposition == 'attachment'
+    cd = parse_headers('attachment; filename="spa  ced";')
     assert cd.filename_unsafe == u'spa ced'
 
 
-
-def test_roundtrip():
+@pytest.mark.parametrize("name", [
+    'a b', 'a b ', ' a b', 'a\"b', u'aéio o♥u"qfsdf!'
```
Comment (homm) [inline test_rfc6266_parser.py:57]: no `'a   b'`?

File: test_rfc6266_parser.py [hunk 5]
```diff
@@ -56,42 +45,25 @@ def test_location_fallback():
...
     def roundtrip(filename):
-        return parse_headers(build_header(filename)).filename_unsafe
+        header = build_header(filename)
+        if sys.version_info >= (3,):
+            header = header.decode()
+
+        return parse_headers(header).filename_unsafe
 
     def assert_roundtrip(filename):
         assert roundtrip(filename) == filename
```
Comment (homm) [inline test_rfc6266_parser.py]:
```suggestion
    assert roundtrip(filename) == filename
```

File: rfc6266_parser.py [hunk 6]
```diff
@@ -168,97 +158,34 @@ def ensure_charset(text, encoding):
         return text
 
 
-def parse_headers(content_disposition, location=None, relaxed=False):
+def parse_headers(content_disposition, location=None):
```
Comment (homm) [inline rfc6266_parser.py:151]: Add content_disposition type and rtype

**Review by homm — CHANGES_REQUESTED** [review]

File: test_rfc6266_parser.py (hunk 4 above)
Comment (genme) [reply test_rfc6266_parser.py:57]: It was use case related to strict mode (`relaxed=False`).

File: setup.py [hunk 7]
```diff
@@ -2,21 +2,29 @@
...
     keywords='rfc6266 parser Content-Disposition http attachments',
     name='rfc6266-parser',
-    version='0.0.6',  # semver
+    version='1.0.0',
     license='GNU LGPL',
     platforms='OS-independent',
     py_modules=['rfc6266_parser', 'test_rfc6266_parser'],
-    install_requires=['LEPL'],
-    use_2to3=True,
+    install_requires=['werkzeug==1.0.1'],
```
Comment (homm) [inline setup.py]: Relax it

File: rfc6266_parser.py [hunk 8]
```diff
@@ -168,97 +156,36 @@ def ensure_charset(text, encoding):
...
     """Build a ContentDisposition from header values.
+    :type content_disposition: str|unicode|None
```
Comment (homm) [inline rfc6266_parser.py]: `build_header` returns bytes, while `parse_headers` receives text. This is asymmetrical.

File: test_rfc6266_parser.py [hunk 9]
```diff
@@ -56,42 +47,22 @@ def test_location_fallback():
...
+])
+def test_roundtrip(name):
     def roundtrip(filename):
-        return parse_headers(build_header(filename)).filename_unsafe
-
-    def assert_roundtrip(filename):
-        assert roundtrip(filename) == filename
+        header = build_header(filename)
+        if sys.version_info >= (3,):
+            header = header.decode()
```
Comment (homm) [inline test_rfc6266_parser.py]: There is shouldn't be such branching, types should be consistent over different versions.

**Review by homm — APPROVED** [review]

**Pull request merged**